### ISSUE TITLE
Command Line Installation Compatibility

### ISSUE THREAD

**Issue by brittonhayes** [princjef/gomarkdoc#31]
Hey there friend 👋

Love the tool. Just noticed that the installation command of `go get -u github.com/princejef/gomarkdoc/cmd/gomarkdoc` appears to get mad at BlackFriday v2 being missing.

However if I run `GO111MODULE=on go get -u github.com/princejef/gomarkdoc/cmd/gomarkdoc`, the installation works great and I'm up and running with your tool! Could be a Black Friday issue, but wanted to give heads up.

Tldr; just in case other people run into the same issue, would you be able to update the readme with the GOMODULE-prepended install command?

**Comment by princjef** [comment]
Good catch @brittonhayes! This most likely was introduced when we moved over to the go modules version of blackfriday a little while back. Looks like that broke the behavior on install.

I'll update the command with the extra `GO111MODULE` environment variable to clear up the confusion!
